app/src/main/java/com/nervousfish/nervousfish/modules/constants/Constants.java

```diff
@@ -37,6 +39,10 @@
      */
     private Constants(final IServiceLocator serviceLocator) {
         this.androidFilesDir = serviceLocator.getAndroidFilesDir();
+        //noinspection StringConcatenationMissingWhitespace because this is a file path
+        this.databaseContactsPath = this.androidFilesDir + Constants.DB_CONTACTS_PATH;
+        //noinspection StringConcatenationMissingWhitespace because this is a file path
+        this.databaseUserPath = this.androidFilesDir + Constants.DB_USERDATA_PATH;
```

Comment:
I'm against this change, especially since you're now also writing this concatenation in the `readObject()` method (see [here](https://github.com/ericcornelissen/NervousFish/blob/2a4aabfecd1c3391936a0ce27a8dece5091ccd14/app/src/main/java/com/nervousfish/nervousfish/modules/constants/Constants.java#L101))